hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/service/TestKeyDeletingService.java

```diff
@@ -274,9 +271,9 @@ public void checkDeletionForPartiallyCommitKey()
           try {
             return keyManager.getPendingDeletionKeys(Integer.MAX_VALUE)
                 .stream()
-                .map(BlockGroup::getBlockIDList)
-                .flatMap(Collection::stream)
-                .collect(Collectors.toList()).size() == 3;
+                .mapToInt(e -> e.getKeyLocationVersions().stream()
+                    .map(OmKeyLocationInfoGroup::getLocationList).
+                    mapToInt(List::size).sum()).sum() == 3;
           } catch (IOException e) {
```

Comment:
Can we also add a unit test to verify updation of the usedBytes and namespace after key cleanup.